src/filters/qualitative-filter.js

```diff
@@ -233,7 +233,6 @@ const QualitativeFilter = ({ column: { filterValue, preFilteredRows, setFilter,
                   onClick={(e) => {
                     handleExceptOnClick(e, opt)
                   }}
-                  disableRipple={true}
```

Comment:
i meant if it's intended to be `true` then it can be specified as

```jsx
<Button
  ...
  disableRipple
>
```

which is equivalent to

```jsx
<Button
  ...
  disableRipple={true}
>
```